fe/fe-core/src/main/java/com/starrocks/catalog/Database.java

```diff
@@ -137,7 +148,14 @@ public void readUnlock() {
     }
 
     public void writeLock() {
+        long startMs = System.nanoTime() / 1000000;
         this.rwLock.writeLock().lock();
+        long endMs = System.nanoTime() / 1000000;
+        if (endMs - startMs > SLOW_LOCK_MS && endMs > lastSlowLockLogTime + SLOW_LOCK_LOG_EVERY_MS) {
+            lastSlowLockLogTime = endMs;
+            LOG.warn("slow write lock db:" + id + " " + fullQualifiedName + " " + (endMs - startMs) + "ms",
+                    new Exception());
+        }
```

Comment:
these two function is so same, better to make it a common function.

Reply:
It's a cherry-pick, I will try to refactor it in another PR.